DesktopEdge/Models/ZitiIdentity.cs

```diff
@@ -147,7 +147,7 @@ public static ZitiIdentity FromClient(DataStructures.Identity id) {
 #if DEBUG
             zid.MFADebug("002");
 #endif
-            if (id.Services != null) {
+            if (id.Services != null && id.Services.Count > 0) {
```

Comment:
This bug fix addresses a situation where an identity is enrolled, but disabled.  At startup, the identity is assessed to ascertain if there are failing posture checks.  The existing check ensure that identities which has "null" services were bypassed, however if the identity DID have services, yet is simply turned off/disabled at the moment with "0" services bound to it, the check would pass.  The result is that an MFA icon lock would appear which could not be removed without enabled the identity and restarting the tunnel.  This addresses the missing check and ensures that the identity reports at least 1 available service before checking further in the code 
![Screenshot 2024-11-15 at 12 21 25 PM](https://github.com/user-attachments/assets/78ed6f59-6603-4ef7-8ccb-ccd351f4b46a)
block.